travis-ymls/pipeline.travis.yml

```diff
@@ -2,7 +2,7 @@
 #
 # Package             : pipeline
 # Source Repo         : https://github.com/tektoncd/pipeline.git
-# Travis Job Link     : https://travis-ci.com/github/dthadi3/pipeline/builds/217320099
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/pipeline/builds/217856077
```

Comment:
I see some failures in the test logs such as "TestFetch".

Reply:
As per my assumption it is checking for git which got failed status and remaining everything is same as intel log only.
Any additional inputs on this @gururajrkatti  @seth-priya